### ISSUE TITLE
Cursorposition nach CMD-X

### ISSUE THREAD

**Issue by fernsehmuell** [Ultraschall/ultraschall-portable#55]
Derzeit bleibt die Cursorposition nach dem Ripple-Cut einfach gleich.
Vorschlag: wenn der Cursor im zu löschenden Bereich liegt, sollte nach dem Cut der Cursor an die Schnittstelle springen, damit man dort gleich weiterarbeiten kann.

**Comment by rstockm** [comment]
Hm, die jetzige Implementierung löst leider nur ein Teilproblem. Wenn man "normal" mit der Trim-Funktion arbeitet, sprich einen Bereich aufzieht, mit `P` vorhört und dann schneidet, klebt der Edit-Cursor ohnehin die ganze Zeit an der linken Auswahlkante.
Dein Fix hilft also bei einem edge-case, nicht aber bei der größeren Macke: wenn ich etwa mit `P` vorgehört habe, und zufrieden bin, und die Wiedergabe einfach weiterlaufen lasse, dann "springt" der Play-Cursor um die weite des Schnittes nach rechts und überspringt einfach ungefragt Audio.
Lösung: man misst die Breite des Schnittes aus (Selection) und verschiebt nach dem Schnitt den Play-Cursor (!) um diesen Betrag nach links, wenn er sich rechts vom Schnitt befindet.

**Comment by rstockm** [comment]
Ja, das sieht gut aus. Einen neuen Edge-Case habe ich gerade noch gefunden, den wir zumindest diskutieren sollten:
Wenn der Follow-Mode aktiv ist, und ich dennoch links etwas mit cmd+x schneide, löuft das soweit alles prima, der Follow-Mode wird jedoch deaktiviert.
Ich denke: wenn Play etc. weiter läuft wäre es eigentlich erwartungskonform, wenn auch der Follow-Modus wieder aktiv wäre?

**Comment by fernsehmuell** [comment]
Habe ich geändert. Der Followmode bleibt jetzt an.
